client/my-sites/site-settings/jetpack-site-stats.jsx

```diff
@@ -171,6 +172,10 @@ class JetpackSiteStats extends Component {
 						}
 					</FormFieldset>
 				</FoldableCard>
+
+				<CompactCard href={ '/stats/day/' + siteId }>
+					{ translate( 'View your stats' ) }
```

Comment:
Hi! I've found a possible matching string that has already been translated **13** times:
`translate( 'View your site stats' )` *ES Score: 6*

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).